mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MailboxMapperACLTest.java

```diff
@@ -40,6 +40,8 @@
 
 public abstract class MailboxMapperACLTest {
     private static final long UID_VALIDITY = 42;
+    public static final boolean POSITIVE = true;
+    public static final boolean NEGATIVE = !POSITIVE;
```

Comment:
`false` is better

Reply:
probably, but that's just a revert, I prefer to not modify non concerned lines, this PR in big enough.